concrete/src/Page/Collection/Collection.php

```diff
@@ -728,124 +737,34 @@ public function resetAreaCustomStyle($area)
      *
      * @return string|null
      */
-public function outputCustomStyleHeaderItems($return = false)
+    public function outputCustomStyleHeaderItems($return = false)
     {
-        if (!Config::get('concrete.design.enable_custom')) {
-            return '';
+        $app = Application::getFacadeApplication();
+        if (!$app['config']->get('concrete.design.enable_custom')) {
+            return $return ? '' : null;
         }
 
         $psss = [];
-        $txt = Loader::helper('text');
-        CacheLocal::set('pssCheck', $this->getCollectionID() . ':' . $this->getVersionID(), true);
```

Comment:
These local cache values are not used since `getEntry` removed in https://github.com/concretecms/concretecms/commit/5a164293616e17ea3d5130285b699a88fffc5b19